sdk/src/main/scala/com/stratio/sparkta/sdk/exception/MockException.scala

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (C) 2014 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stratio.sparkta.sdk.exception
+
+/**
+ * Created by gschiavon on 14/05/15.
```

Comment:
You should write a description with instructions of how to use this exception and use "@auhor gschiavon" annotation instead of "Created by ..."